src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -905,6 +905,9 @@
   - name: --max-count
     type: int
     short-summary: Maximum nodes count used for autoscaler, when "--enable-cluster-autoscaler" specified. Please specify the value in the range of [1, 100]
+  - name: --scale-down-mode
+    type: string
+    short-summary: "Describes how VMs are added to or removed from nodepools."
```

Comment:
Please use the first person voice
```suggestion
    short-summary: "Describe how VMs are added to or removed from nodepools."
```

Reply:
it should be "Describes". Short for "The parameter describes how VMs are added to or removed from nodepools"